### ISSUE TITLE
any alternatives? :(

### ISSUE THREAD

**Issue by fvnky07** [Tagggar/Firefox-Alpha#26]

**Comment by suchacat** [comment]
Hello, I have made a fork that makes the theme work except the search results are not able to be clicked

**Comment by mebishp** [comment]
I saw your fork and its actually better than the og but not perfect

**Comment by fvnky07** [comment]
Ima have to check it out 😁

**Comment by suchacat** [comment]
> Ima have to check it out 😁

How did it go?

**Comment by fvnky07** [comment]
> > Ima have to check it out 😁
> 
> How did it go?

I am trying it now. i'll let you know

**Comment by suchacat** [comment]
> 
> 
> > > Ima have to check it out 😁
> 
> > 
> 
> > How did it go?
> 
> 
> 
> I am trying it now. i'll let you know
> 
> 

Alright 😁

**Comment by fvnky07** [comment]
> Alright 😁

yeah it works! thanks for the forking it. I was never bothered to haha. how did you fix it? Theres some errors but I dont know if thats from the original plugin. Its perfectly usable. Ive been looking for a minimal firefox theme forever. 

thanks again :D

**Comment by fvnky07** [comment]
my only gripe would be this issue that there is spacing where the titlebar buttons used to be
![image](https://github.com/Tagggar/Firefox-Alpha/assets/76881322/75f668b8-4d06-4937-9f55-7b32a86a124d)

**Comment by suchacat** [comment]
> > Alright 😁
> 
> yeah it works! thanks for the forking it. I was never bothered to haha. how did you fix it? Theres some errors but I dont know if thats from the original plugin. Its perfectly usable. Ive been looking for a minimal firefox theme forever.
> 
> thanks again :D

No problem! It was just a matter of updating the code that he took from MrOtherGuy firefox hacks and making the downloads visible

**Comment by suchacat** [comment]
> my only gripe would be this issue that there is spacing where the titlebar buttons used to be ![image](https://private-user-images.githubusercontent.com/76881322/333938194-75f668b8-4d06-4937-9f55-7b32a86a124d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY3NzcxNzgsIm5iZiI6MTcxNjc3Njg3OCwicGF0aCI6Ii83Njg4MTMyMi8zMzM5MzgxOTQtNzVmNjY4YjgtNGQwNi00OTM3LTlmNTUtN2IzMmE4NmExMjRkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI3VDAyMjc1OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThlNDE5MzEwODJiNTg2MWUxZGJiZTA4NTA0YjlmYTMxMTQ1Y2Y4OWEyYmM0NGUyYzZlMjZmMTM4YTIzYTZmOWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.irMZyFemUPl9T9NbI-raV6RAoLhtF8uiDUD_m7nzcYI)

That's from the added spacing due to the removed titlebar buttons, you can lessen the space by changing `margin-right: 112px !important;` on line 442

**Comment by fvnky07** [comment]
i see, that worked! 

It would be nice if i could enable the addon button, so i can quickly toggle addons.

**Comment by suchacat** [comment]
Hello!, to enable the add-on button remove line 440 and lines 53-54

**Comment by diip3sh** [comment]
I have use to css from @suchacat  but there is one issue with when I open new tab the url search input alignment is bit down to other tabs and I can't figure out what was causing it can some help me plz

**Comment by suchacat** [comment]
> I have use to css from @suchacat  but there is one issue with when I open new tab the url search input alignment is bit down to other tabs and I can't figure out what was causing it can some help me plz

Hello, I have encountered this during my testing as well and it seems to only happen on windows with no fix.

**Comment by KiKaraage** [comment]
Hi @suchacat, I just applied your fork and sadly clicking on an active tab doesn't make URL bar showed up; I had to use `Ctrl + L` for it. Any possible fix?

(Also, I want to know how to show full tab title when hovered. I haven't touch CSS for quite some time, so I didn't know which line should I modify for that.)

**Comment by emslima** [comment]
Any new about it?

**Comment by KiKaraage** [comment]
> Any new about it?

Try my fork! It supported Firefox v133

**Comment by emslima** [comment]
> > Any new about it?
> 
> Try my fork! It supported Firefox v133

Not working, theres is any tip or documentation online to follow and fix that? It would be great for me learn and fix
